### PR TITLE
Added login system (part 2)

### LINKED ISSUE #25: As an Admin I want to login so I have access to the admin panel.
**Acceptance criteria:**
Given: I am on the dashboard page 
When: I click the login button in the navigation bar 
Then: I input my username and password in a pop-up window 
When:  The entered credentials are correct and validated 
Then: I will see the admin panel 
When: The entered credentials are incorrect or invalid 
Then: I prompt the user to retry and provide additional information about what went wrong

### LINKED ISSUE #26: As an Admin I want to hide the admin panel from non-admin users so only the admin can access it.
**Acceptance criteria:**
GIVEN On the dashboard,
WHEN Logged in as an admin,
THEN The admin can see the button for the admin panel,
WHEN User is not an admin or is not logged in,
THEN There will not be a button for the admin panel.

### LINKED ISSUE #56: As a User I want to hide the blog panel from guests so only verified users can see it.
Acceptance criteria:
GIVEN: On the dashboard,
WHEN: Logged in as an user and verified,
THEN: The user can see the button for the blog panel,
WHEN: User is not logged in or not verified.
THEN There will not be a button visible for the blog panel.

### LINKED ISSUE #57: As a User I want to verify my email so that I can have access to create a blog.
Acceptance criteria:
GIVEN: On the register page.
WHEN: I fill in my information and click on register.
THEN: I will receive an email to verify my account.
WHEN: I click verify in this email.
THEN: I will have access to my account.

### PR DESCRIPTION
**Pull request by koenhamstra**
Quinten and I made the following:

- email authentication.
- email verification.
- created view for email verification.
- created view for email authentication.
- created admin hotmail (details will be on onedrive).

**To make the email verification work you have to add the admin details from above into your .env file**

### CODE REVIEW

File: public/css/app.css [hunk 1]
```diff
@@ -1,3 +1,12 @@
+/* verify email page*/
+.verify-message{
```
Comment (avanheeren) [inline public/css/app.css:2]: no space after .verify-message

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -1,68 +1,58 @@
...
+            <!-- Validation Errors -->
+            <x-auth-validation-errors class="mb-4" :errors="$errors" />
+
+            <form method="POST" action="{{ route('register') }}">
+                @csrf
+
+                <!-- Name -->
+                <div>
+                    <x-label for="name" :value="__('Name')" />
+
```
Comment (avanheeren) [inline resources/views/auth/register.blade.php:17]: whitespace?

File: resources/views/auth/register.blade.php [hunk 3]
```diff
@@ -1,68 +1,58 @@
...
+                    <x-input id="name" class="block mt-1 w-full" type="text" name="name" :value="old('name')" required
+                        autofocus />
+                </div>
+
+                <!-- Email Address -->
+                <div class="mt-4">
+                    <x-label for="email" :value="__('Email')" />
+
```
Comment (avanheeren) [inline resources/views/auth/register.blade.php:25]: whitespace

File: resources/views/verified-email.blade.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+<section class="hero  is-medium  is-bold">
+    <div class="header-img">
+        <img src="./image/sdg-header-en.png" />
+    </div>
+</section>
+<article class="message is-info verify-message">
+    <div class="message-body is-size-3">
+    Welcome... you have verified your email.
+    You are now able to use your account.
+  </article>
```
Comment (avanheeren) [inline resources/views/verified-email.blade.php:13]: indent isnt even

**Review by avanheeren — CHANGES_REQUESTED** [review]
whitespace and indenting

**Review by awqfrxx — APPROVED** [review]
Woah who made this? Crazy code!

File: resources/views/common/navbar.blade.php [hunk 5]
```diff
@@ -2,7 +2,7 @@
     <div class="container">
         <div class="navbar-brand">
             <a href="/" class="navbar-item">
-                <img class="logo" src="/image/HZ_logo.png"/>
+                <img class="logo" src="/image/HZ_logo.png" />
```
Comment (Koreyoshi159) [inline resources/views/common/navbar.blade.php]: ??

File: resources/views/welcome.blade.php [hunk 6]
```diff
@@ -7,14 +7,23 @@
         </div>
     </section>
 
+    <section class="hero  is-medium  is-bold">
```
Comment (Koreyoshi159) [inline resources/views/welcome.blade.php:10]: Where is section closed ?

**Pull request merged**